internal/isoeditor/rhcos_test.go

```diff
@@ -135,56 +125,43 @@ var _ = Context("with test files", func() {
 			isoPath, err := editor.CreateMinimalISOTemplate(defaultTestServiceBaseURL)
 			Expect(err).ToNot(HaveOccurred())
 
-			// Open image and read header
-			iso, err := os.OpenFile(isoPath, os.O_RDONLY, 0o664)
-			Expect(err).ToNot(HaveOccurred())
-			header := make([]byte, int64(32768))
-			_, err = iso.Read(header)
-			Expect(err).ToNot(HaveOccurred())
-
 			// Read offsets
-
-			headerBytes := header[len(header)-24:]
-			buf := bytes.NewReader(headerBytes)
-			offsetInfo := new(OffsetInfo)
-			err = binary.Read(buf, binary.LittleEndian, offsetInfo)
+			ignitionOffsetInfo, ramDiskOffsetInfo, err := readHeader(isoPath)
```

Comment:
This is kinda weird to me. This is a test for `CreateMinimalISOTemplate`, but it's using `readHeader` to validate the result.

What if they're both wrong in complimentary ways? I suppose that doesn't matter because we're the only ones using this .. Maybe it would be worth committing a contrived iso to test these against?

Reply:
Indeed, we might want to push a pre-generated 'dummy' image to the repo and test against it. But then we would have to maintain the generation script and push it for each change, which could be error prone from similar reasons. So I think we can leave it for know and tackle later on if we have a good solution.

Reply:
Yeah, makes sense :+1: